parsers/lib/parsers.py

```diff
@@ -13,6 +13,8 @@
 GENERATION_FORECAST_PARSERS = {}
 EXCHANGE_FORECAST_PARSERS = {}
 PRODUCTION_CAPACITY_PARSERS = {}
+DAYAHEAD_PRICE_PARSERS = {}
+REALTIME_PRICE_PARSERS = {}
```

Comment:
These should be renamed to be LMP specific since we might add other price parsers in the future.